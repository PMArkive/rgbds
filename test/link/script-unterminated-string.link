rom0
	"ROM0
romx 2
	"ROM2 1K
	"ROM2 1
