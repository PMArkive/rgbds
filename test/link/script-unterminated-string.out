error: script-unterminated-string.link(2): Unterminated string
error: script-unterminated-string.link(4): Unterminated string
error: script-unterminated-string.link(5): Unterminated string
Linking failed with 3 errors
